src/command_modules/azure-cli-container/azure/cli/command_modules/container/commands.py

```diff
@@ -27,6 +27,6 @@ def load_command_table(self, _):
         g.custom_command('attach', 'attach_to_container')
 
     with self.command_group('container', container_group_sdk) as g:
-        g.command('restart', 'restart')
+        g.command('restart', 'restart', supports_no_wait=True)
         g.command('stop', 'stop')
```

Comment:
Stop doesn't need a no wait?

Reply:
No. As far as I can tell, it is never long-running.